test/e2e/mock-e2e.js

```diff
@@ -4,6 +4,13 @@ const blacklistedHosts = [
   'mainnet.infura.io',
   'rinkeby.infura.io',
   'ropsten.infura.io',
+  'arbitrum-mainnet.infura.io',
+  'bsc-dataseed.binance.org',
+  'api.avax.network',
+  'optimism-mainnet.infura.io',
+  'polygon-mainnet.infura.io',
+  'api.harmony.one',
+  'palm-mainnet.infura.io',
```

Comment:
Do we need all of these networks or just a few?  Many are unused.

Reply:
Done.